### ISSUE TITLE
Visual focus indicator on textboxes

### ISSUE THREAD

**Issue by AxelWol** [microsoft/fluentui#13400]
### Feature
a)
All input controls have some kind of visual focus indicator. The focus of a selected element is shown as a black thin border line  (e.g. on Radios, Checkboxes) or as a blue box on the search box.
Why do Textboxes do not have a focus indicator? They should have the same blue border as search boxes.

b) An input error is displayed with a red border. That's a good way to display a problem.
But how can we combine an error state with a focus indicator?
One idea: display the red line with a line style pattern (white/red) or display the red boarder of the input control with another tone of red (darker).

#### Component
Modification of Textbox input control visual style on focus.

**Comment by aneeshack4** [comment]
@AxelWol Thanks for reaching out to us. There's a blue box as a focus indicator on the TextField:
![image](https://user-images.githubusercontent.com/4161791/83282787-bb272c00-a19f-11ea-852d-cc5070733a2d.png). For the red border input style - just to give you context, we follow the Fluent Design Toolkit so this is by design. https://www.figma.com/file/F5BN2yodUyHlOfpL4KJCFK/Fluent-Web?node-id=1%3A924&viewport=-917%2C-294%2C0.651712954044342 
![image](https://user-images.githubusercontent.com/4161791/83282937-fa557d00-a19f-11ea-8a81-c4f7552b3454.png)
